sources/ProfilesWindowPreferencesViewController.m

```diff
@@ -119,6 +120,10 @@ - (void)awakeFromNib {
                     key:KEY_PREVENT_TAB
                    type:kPreferenceInfoTypeCheckbox];
 
+   [self defineControl:_defaultBGAlphaOnly
+                   key:KEY_DEFAULT_BG_ALPHA_ONLY
+                  type:kPreferenceInfoTypeCheckbox];
```

Comment:
This should be disabled when transparency is off. Assign the result of this method call to `info` and then do
````
    info.observer = ^() { _defaultBGAlphaOnly.enabled = (_transparency.doubleValue > 0); };
````

(look at how the blur slider gets disabled for what an example of what I mean)